dashboard/back-end/src/config/environments/index.js

```diff
@@ -0,0 +1,30 @@
+const developmentConfig = require("./development");
+const stagingConfig = require("./development");
+const productionConfig = require("./development");
+
+const environment = process.env.NODE_ENV;
```

Comment:
environment variable configuration module. So we can seamlessly use different configurations and variables based on what environment we're in.